.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,48 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone the crud module
+        uses: actions/checkout@v3
+        with:
+          repository: slavakirichenko/crud
```

Comment:
So... when this one will be replaced with the real one?

Reply:
Fixed.